src/styles/table/table.css

```diff
@@ -0,0 +1,40 @@
+@import url(../../themes/themes.css);
+/*
+ *  Styling for native-HTML tables.
+ **/
+
+table {
+  border-collapse: collapse;
+}
+
+tr {
+  background-color: var(--surface);
```

Comment:
I think this would look better as 

   background-color: var(--background);
   border-bottom: 1px solid var(--on-background);

If you that then all the hovers should use var(--surface) as the backgound.

Reply:
Done, but used --surface-1dp for nested hovers.

Reply:
Actually, with the tr background-color not being transparent, nested tables retain the original --background color, so there's no need for any handling of nested hovers.  I did use --surface-2dp instead of --on-background for the border; I wanted a more subtle border to make it easier to read across a line without really noticing it.  I'm also not a huge fan of --surface being so light in dark mode, it seems like there's a much bigger contrast between --background and --surface in dark mode than there is in light mode.

Reply:
Ah, I think when I copied Weston's original work here I messed up the dark colors. I have a local fix for Perf where I set

    --surface: var(--grey-900);
    --surface-1dp: var(--grey-800);
    --surface-2dp: var(--grey-700);

in darkmode. If you agree we should probably push that change back up to elements-sk.

Reply:
I tried locally and that looks much better to me!

Reply:
OK, I'll send a PR for that. I'll also send a second one that creates vars for the opacity percentages you used below.